Project-3/README.md

```diff
@@ -11,67 +11,238 @@ For any question involving the use of Protege, please be sure to import:
+  In order to have a wide ranging and flexible “has role in modeling” object property to classify possible relations between an artifact x and entity y, I argue that it is more advantageous and preferable to unlist the asymmetric characteristic so that possible symmetry relations and relations where it is best to remain agnostic on symmetry/asymmetry assertions may be represented in artifact to entity modeling relations. This is both an appeal to pragmatic and philosophical reasons. Unless all relations between artifact x and entity y are asymmetric, they should not be listed as such. By doing so, other non-asymmetric modeling relations cannot adequately be represented.
+  Thus, although the “has role in modeling” object property has the asymmetric characteristic listed (likely for pragmatic purposes specific to modeling ontologies with distinct ends), I argue that it would be better to dispense with it if one’s aim is to represent a wider domain of possible symmetric/asymmetric relations between artifacts with specific modeling roles and entities. Moreover, a more robust and comprehensive ontology would be representable as a result of unlisting the asymmetric characteristic.
+
   (d)  Functional 
+  The object property, “characteristic of”, should not have the functional property listed.
+  It’s definition states: “a relation between a specifically dependent continuant (the characteristic) and any other entity(the bearer), in which the characteristic depends on the bearer for its existence.” 
+  The functional property relation means that for any given individual, the property can have at most one value. In other words, there can be at most one out going       relationship along the property for that individual. This means that a functional object property has a multiplicity 0..1 attached to it. For example, it might be     said that the continuant red is “characteristic of” Washington Red Delicious Apples. Or put in another way, this red color is a characteristic of this Washington Red   Delicious Apple. 
+  However, it is not the case that a given characteristic necessarily corresponds with at most one value, univocally. It could very well be the case that the red         characteristic of Washington Red Delicious Apples might also have another corresponding value in a red colored car of the same hue. This means that correspondence     between a characteristic and a bearer is not necessarily functional, i.e. has a binary value between 0 and 1 at most. 
```

Comment:
It depends on whether we are talking about a trope or a universal here (metaphysics of property). The redness (as a trope) of this apple can never be identical to the redness (as a trope) of any car, though these two redness tropes may be very similar. A property trope can only be instantiated once.